group_transactions/src/main.rs

```diff
@@ -0,0 +1,49 @@
+use algonaut::algod::v2::Algod;
+use algonaut::core::MicroAlgos;
+use algonaut::transaction::account::Account;
+use algonaut::transaction::tx_group::TxGroup;
+use algonaut::transaction::Pay;
+use algonaut::transaction::TxnBuilder;
+use dotenv::dotenv;
+use std::env;
+use std::error::Error;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn Error>> {
+    // load variables in .env
+    dotenv().ok();
+
+    let algod = Algod::new(&env::var("ALGOD_URL")?, &env::var("ALGOD_TOKEN")?)?;
+
+    let account1 = Account::from_mnemonic("kiss solar urge under dragon alone merry empty monkey hospital crisp over version tragic move cat company climb draft twin cram patrol say able sausage")?;
+    let account2 = Account::from_mnemonic("maze fortune disagree inside sniff mail balcony parade error alert isolate acoustic choose dress music response room forget crash lottery chronic dragon second able mad")?;
+
+    let params = algod.suggested_transaction_params().await?;
+
+    // To keep the example short and as self-contained as possible, both transactions send Algos.
+    // Normally you'll want to submit e.g. a payment and asset transfer or asset transfers for different assets.
```

Comment:
We should try to expand this code to do the group transaction we're interested in, to redeem the asset by transferring it back to the creator and destroying it.